docs/docsite/extra-docs.yml

```
@@ -7,6 +7,7 @@ sections:
   - title: Guides
     toctree:
       - filter_guide
+      - moddev_guide
       - test_guide
```

Maybe split this up into two sections:
```suggestion
  - title: User Guides
    toctree:
      - filter_guide
      - test_guide
  - title: Developer Guides
    toctree:
      - moddev_guide
```
Maybe also move the dev guide to the bottom (after Cloud Guides), since the Cloud Guides are also user guides.

---

That's actually how I first wrote it, but then I thought it would be too much.

I should be going for this $(TZ=NZST tonight).

---

I think if we are making a new section at that level, then maybe I should get rid of the `moddev_guide.rst` file entirely and refer to the `moddev_guide_deps` already at the `extra-docs` level. What do you think?

---

@felixfontein ^ ?

---

Sounds good to me! (Sorry for the slow response, this was a really busy week...)

---

I figured so, no worries. Welcome back! :-)